components/script/dom/canvasrenderingcontext2d.rs

```diff
@@ -800,6 +800,13 @@ impl CanvasRenderingContext2DMethods for CanvasRenderingContext2D {
         receiver.recv().unwrap()
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-context-2d-filltext
+    fn FillText(&self, text: DOMString, x: f64, y: f64, max_width: Option<f64>) {
+        let parsed_text: String = text.parse().unwrap();
```

Comment:
Oh. I'm afraid `DOMString` doesn't have `into()` and I'm not sure how to add it to `components/script/dom/bindings/str.rs`, can you help?